recipes/octomap/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "octomap" %}
+{% set version = "1.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OctoMap/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5f81c9a8cbc9526b2e725251cd3a829e5222a28201b394314002146d8b9214dd
+  patches:
+    # Compatibility with C++17
+    - byte.patch
+    - fix-exported-targets.patch  # [win]
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
```

@wolfv In that case, how about:
```suggestion
    # Only static libraries are built on Windows
    - {{ pin_subpackage(name, max_pin='x.x') }}  # [not win]
```